src/com_tjvendors/admin/tjvendorsTemplate.json

```diff
@@ -218,27 +288,41 @@
     "client": "com_tjvendors",
     "key": "editVendorMailToAdmin",
     "title": "Edit Vendor profile mail to admin",
-    "email_body": "<p>Dear Admin,<br /><br />The user {vendorer.name} has made changes to vendor <strong>{info.vendor_title}</strong>.<br /><br />Vendor details are -<br /><strong>Vendor title</strong> - {info.vendor_title}<br /><strong>User</strong> - {vendorer.name}<br /><strong>Client</strong> - {info.vendorClient}<br /><br />This is system generated email, please do not reply. Many thanks from {info.sitename}</p>",
-    "sms_body": "",
-    "push_body": "",
-    "web_body": "",
-    "email_subject": "Vendor \"{info.vendor_title}\" has been edited on {info.sitename}",
-    "sms_subject": "",
-    "push_subject": "",
-    "web_subject": "",
-    "email_status": 1,
-    "sms_status": 0,
-    "push_status": 0,
-    "web_status": 0,
     "user_control": 1,
+    "state": 1,
     "core": 1,
+    "email":{
+		"state": 1,
+		"cc": "",
+		"bcc": "",
+		"from_name": "",
+		"from_email": "",
+		"emailfields": {
+			"emailfields0": {
+				"id": "",
+				"language": "*",
+				"subject": "Vendor \"{{info.vendor_title}}\" has been edited on {{info.sitename}}",
+				"body": "<p>Dear Admin,<br /><br />The user {{vendorer.name}} has made changes to vendor <strong>{{info.vendor_title}}</strong>.<br /><br />Vendor details are -<br /><strong>Vendor title</strong> - {{info.vendor_title}}<br /><strong>User</strong> - {{vendorer.name}}<br /><strong>Client</strong> - {{info.vendorClient}}<br /><br />This is system generated email, please do not reply. Many thanks from {{info.sitename}}</p>"
+			}
+		}
+	},
+	"sms": {
+		"state": 1,
+		"smsfields": {
+			"smsfields0": {
+				"id": "",
+				"language": "*",
+				"body": "Dear Admin, The user {{vendorer.name}} has made changes to vendor {{info.vendor_title}}. Vendor details are - Vendor title - {{info.vendor_title}} User - {{vendorer.name}} Client - {{info.vendorClient}}"
```

Comment:
change as


Dear Admin, The user {{vendorer.name}} has made changes to vendor titled {{info.vendor_title}} client {{info.vendorClient}}